### PR TITLE
FB - finalize git integration (screenshot + some content)

### PR DESCRIPTION
**Pull request by fabienOVH**

### CODE REVIEW

File: pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: "Configurer et utiliser Git avec son hébergement web OVHcloud"
 excerpt: "Découvrez comment configurer et utiliser Git avec votre hébérgement web dans votre espace client OVHcloud"
-updated: 2024-07-01
+updated: 2024-07-18
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md]:
```suggestion
updated: 2024-07-25
```

File: pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md [hunk 2]
```diff
@@ -28,11 +28,11 @@ Dans le paysage numérique actuel, les sociétés sont de plus en plus dynamique
 > Pour l'association et la configuration de Git, vous devrez effectuer des modifications dans votre compte Github. Avant de commencer le guide, connectez-vous à votre compte Github.
 >
 
-### Associer un répertoire à Git
+### Associer un répertoire à Git <a name="associateGitRepo"></a>
 
 > [!warning]
 >
-> Lorsque vous associez un répertoire à Git, tous les domaines présents sur ce répertoire sont également associés à Git. Par exemple, si le répertoire correspondant au site web que vous associez est `www`{.action}, alors tous les domaines associés au répertoire www seront associés à Git.
+> Lorsque vous associez un répertoire à Git, tous les domaines présents sur ce répertoire sont également associés à Git. Par exemple, si le répertoire correspondant au site web que vous associez est `www`, alors tous les domaines associés au répertoire `www` seront associés à Git.
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md]:
```suggestion
> Lorsque vous associez un répertoire à Git, tous les noms de domaine présents sur ce répertoire seront également associés à Git. Par exemple, si le répertoire correspondant au site web que vous associez est `www`, alors tous les noms de domaine associés au répertoire `www` seront également associés à Git.
```

File: pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md [hunk 3]
```diff
@@ -43,27 +43,23 @@ Connectez-vous à votre [espace client OVHcloud](/links/manager) et effectuez le
...
+- Branche du dépôt Github
 - Webhook (optionnel)
 
-#### Associer une clé SSH à Github
+#### Associer une clé SSH à Github <a name="linkSSHKey"></a>
 
 > [!primary]
 >
-> La génération d'une clé SSH est une étape cruciale, car elle établit une connexion sécurisée et cryptée entre le répertoire de votre site web et le dépôt Git. Cette clé assure que les transferts de données et les modifications de code se font de manière sûre et authentifiée, prévenant les accès non autorisés et garantissant l'intégrité du code.
+> La génération d'une clé SSH est une étape cruciale, car elle établit une connexion sécurisée et cryptée entre le répertoire de votre site web et le dépôt GitHub. Cette clé assure que les transferts de données et les modifications de code se font de manière sûre et authentifiée, prévenant les accès non autorisés et garantissant l'intégrité du code.
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md]:
```suggestion
> La génération d'une clé SSH est une étape cruciale, car elle établit une connexion sécurisée et chiffrée entre le répertoire de votre site web et le dépôt GitHub. Cette clé assure que les transferts de données et les modifications de code se font de manière sûre et authentifiée, prévenant les accès non autorisés et garantissant l'intégrité du code.
```

File: pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md [hunk 4]
```diff
@@ -77,17 +73,24 @@ Le formulaire d'ajout d'une nouvelle clé SSH s'affiche :
...
+Copiez l'adresse de votre dépôt Github. Celle-ci doit être de la forme `https://github.com/<username>/<repository_name>`{.action}. Retournez sur le formulaire d'association de Git et collez l'adresse de votre dépôt GitHub dans le champ `Dépôt`{.action}. Si le format de l'adresse n'est pas correct, le message d'erreur suivant apparaît.
 
---Screenshot placeholder--
+![Multisite](/pages/assets/screens/control_panel/product-selection/web-cloud/web-hosting/multisite/error-wrong-git-branch-name.png){.thumbnail}
 
-Vous devez maintenant définir la branche de votre dépôt GitHub. La branche par défaut est « Main, mais si vous voulez utiliser une autre branche, créez-en une sur Github en suivant les étapes suivantes :
+Vous devez maintenant définir la branche de votre dépôt GitHub. La branche par défaut est `main`, mais si vous voulez utiliser une autre branche, créez-en une sur Github en suivant les étapes suivantes :
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md]:
```suggestion
Définissez maintenant la branche de votre dépôt GitHub. La branche par défaut est `main`, mais si vous voulez utiliser une autre branche, créez-en une sur Github en suivant les étapes suivantes :
```

File: pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md [hunk 5]
```diff
@@ -108,157 +111,148 @@ Avant de valider le formulaire d'association de Git, assurez-vous que :
...
+Une fois le(s) erreur(s) identifiée(s), associez Git à nouveau. Recommencez l'opération en cliquant sur `...`{.action} de la ligne correspondante, puis sur `Associer Git`{.action}.
 
-Pour résoudre l'erreur liée à la clé SSH, dirigez-vous sur l'onglet `Multisite`{.action}, identifiez la ligne correspondant au répertoire que vous voulez associer à Git, cliquez sur `...`{.action} puis sélectionnez `Configurer Git`{.action}.
+### Déployer votre dépôt Github sur votre hébergement web OVHcloud
 
---Screenshot placeholder--
+Dans votre espace client OVHcloud, accédez à l'onglet `Web Cloud`{.action}, puis cliquez sur `Multisite`{.action}. Dans le tableau qui s'affiche, identifiez la ligne correspondant au domaine que vous souhaitez déployer avec Git. Assurez-vous que le statut de la colonne Git soit `Activé`{.action}. Cliquez sur `...`{.action} puis sur `Déployer Git`{.action}.
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md]:
```suggestion
Dans votre espace client OVHcloud, accédez à l'onglet `Web Cloud`{.action}, puis cliquez sur l'onglet `Multisite`{.action}. Dans le tableau qui s'affiche, identifiez la ligne correspondant au nom de domaine que vous souhaitez déployer avec Git. Assurez-vous que le statut de la colonne Git soit `Activé`{.action}. Cliquez sur le bouton `...`{.action} puis sur `Déployer Git`{.action}.
```

File: pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md [hunk 6]
```diff
@@ -108,157 +111,148 @@ Avant de valider le formulaire d'association de Git, assurez-vous que :
...
 
-Après avoir validé le formulaire s'association de Git, dirigez-vous sur l'onglet `Multisite`{.action}. Une bannière verte vous indique que Git est en cours d'activation. Cela signifie qu'aucune erreur liée au dépôt Git n'a été détectée. Dans le tableau, identifiez les lignes correspondant au répertoire que vous voulez associer à Git. Pour chacune des lignes concernées, dans la colonne `Git`{.action}, la mention `OnGoing`{.action} indique que Git est en cours d'activation.
+![Multisite](/pages/assets/screens/control_panel/product-selection/web-cloud/web-hosting/multisite/info-last-deployment-button.png){.thumbnail}
 
-Pendant le processus d'association de Git, si le système détecte une erreur liée à la clé SSH, alors le statut `Initial error`{.action} apparaît dans la colonne `Git`{.action} du tableau. Pour en savoir plus sur l'erreur, cliquez sur `...`{.action} de la ligne correspondante puis sur `Informations du dernier déploiement`{.action}.
+Une fois le(s) erreur(s) identifiée(s), associez Git à nouveau. Recommencez l'opération en cliquant sur `...`{.action} de la ligne correspondante, puis sur `Associer Git`{.action}.
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md]:
```suggestion
Une fois le(s) erreur(s) identifiée(s), associez Git à nouveau. Recommencez l'opération en cliquant sur le bouton `...`{.action} de la ligne correspondante, puis sur `Associer Git`{.action}.
```

File: pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md [hunk 7]
```diff
@@ -108,157 +111,148 @@ Avant de valider le formulaire d'association de Git, assurez-vous que :
...
+> [!warning]
+>
+> Pour éviter de perdre vos modifications locales, pensez à les enregistrer avant de les écraser par les modifications de la branche distante.
+>
 
---Screenshot placeholder--
+La nouvelle version de votre site web a bien été déployée sur votre hébergement OVHcloud. Si d'autres personnes travaillent sur le même projet et qu'ils apportent des modifications sur le dépôt Github, alors vous pouvez [configurer un webhook sur GitHub](#configureWebhook) pour que leurs modifications soient déployées automatiquement sur votre hébergement web. Cela vous évite de déployer Git manuellement, et votre hébergement web restera toujours à jour.
 
-Au-dessus du tableau, un message de succès s'affiche vous indiquant que le repository est déployé.
+### Modifier un domaine
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md]:
```suggestion
### Modifier un nom de domaine
```

File: pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md [hunk 8]
```diff
@@ -108,157 +111,148 @@ Avant de valider le formulaire d'association de Git, assurez-vous que :
...
 
---Screenshot placeholder--
+Dans votre espace client OVHcloud, accédez à l'onglet `Web Cloud`{.action}, puis cliquez sur `Multisite`{.action}. Dans le tableau qui s'affiche, identifiez la ligne correspondant au domaine que vous souhaitez modifier. Cliquez sur `...`{.action} puis sur `Modifier le domaine`{.action}. Deux scénarios sont possibles :
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md]:
```suggestion
Dans votre espace client OVHcloud, accédez à l'onglet `Web Cloud`{.action}, puis cliquez sur `Multisite`{.action}. Dans le tableau qui s'affiche, identifiez la ligne correspondant au domaine que vous souhaitez modifier. Cliquez sur le bouton `...`{.action} puis sur `Modifier le domaine`{.action}. Deux scénarios sont possibles :
```

File: pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md [hunk 9]
```diff
@@ -108,157 +111,148 @@ Avant de valider le formulaire d'association de Git, assurez-vous que :
...
+Une deuxième fenêtre de confirmation s'affiche avec le récapitulatif de vos changements.
 
-> [!warning]
->
-> Pour éviter de perdre vos modifications locales, pensez à les enregistrer avant de les écraser par les modifications de la branche distante.
->
+![Multisite](/pages/assets/screens/control_panel/product-selection/web-cloud/web-hosting/multisite/modify-domain-step2.png){.thumbnail}
 
-### Modifier un domaine
+Cliquez sur `Confirmer`{.action} pour validez les modifications de votre domaine.
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md]:
```suggestion
Cliquez sur `Confirmer`{.action} pour validez les modifications de votre nom de domaine.
```

File: pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md [hunk 10]
```diff
@@ -108,157 +111,148 @@ Avant de valider le formulaire d'association de Git, assurez-vous que :
...
 
-Dans votre espace client OVHcloud, accédez à l'onglet `Web Cloud`{.action}, puis cliquez sur `Multisite`{.action}. Dans le tableau qui s'affiche, identifiez la ligne correspondant au domaine que vous souhaitez modifier. Cliquez sur `...`{.action} puis sur `Détacher le domaine`{.action}. Deux scénarios sont possibles :
+#### Le domaine est le seul attaché au répertoire
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md]:
```suggestion
#### Le nom de domaine est le seul attaché au répertoire
```

File: pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md [hunk 11]
```diff
@@ -108,157 +111,148 @@ Avant de valider le formulaire d'association de Git, assurez-vous que :
...
 
-- Si votre domaine est le seul du répertoire associé à Git, alors le détachement entrainera la suppression de l'association Git pour le domaine et le répertoire.
-- Si d'autres domaines sont rattachés au même répertoire, alors le détachement entrainera la suppression de l'association Git pour ce domaine.
+La fenêtre suivante s'affiche.
 
-#### Le domaine n'est pas le seul du répertoire
+![Multisite](/pages/assets/screens/control_panel/product-selection/web-cloud/web-hosting/multisite/modify-alone-domain-step1.png){.thumbnail}
 
-Même processus que pour détachement du domaine. En attente des maquettes.
+Comme le message l'indique, [supprimez votre association Git](#deleteGitAssociation) dans un premier temps avant de modifier votre domaine.
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md]:
```suggestion
Comme le message l'indique, [supprimez votre association Git](#deleteGitAssociation) dans un premier temps avant de modifier votre nom de domaine.
```

File: pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md [hunk 12]
```diff
@@ -108,157 +111,148 @@ Avant de valider le formulaire d'association de Git, assurez-vous que :
...
 
-#### Le domaine est le seul du répertoire
+### Détacher un domaine
 
-Même processus que pour détachement du domaine. En attente des maquettes.
+Dans votre espace client OVHcloud, accédez à l'onglet `Web Cloud`{.action}, puis cliquez sur `Multisite`{.action}. Dans le tableau qui s'affiche, identifiez la ligne correspondant au domaine que vous souhaitez détacher de votre hébergement web OVHcloud. Cliquez sur `...`{.action} puis sur `Détacher le domaine`{.action}. Deux scénarios sont possibles :
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md]:
```suggestion
Dans votre espace client OVHcloud, accédez à l'onglet `Web Cloud`{.action}, puis cliquez sur `Multisite`{.action}. Dans le tableau qui s'affiche, identifiez la ligne correspondant au domaine que vous souhaitez détacher de votre hébergement web OVHcloud. Cliquez sur le bouton `...`{.action} puis sur `Détacher le nom de domaine`{.action}. Deux scénarios sont possibles :
```

File: pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md [hunk 13]
```diff
@@ -108,157 +111,148 @@ Avant de valider le formulaire d'association de Git, assurez-vous que :
...
+La fenêtre suivante s'affiche.
 
-#### Le domaine n'est pas le seul du répertoire
+![Multisite](/pages/assets/screens/control_panel/product-selection/web-cloud/web-hosting/multisite/modify-alone-domain-step1.png){.thumbnail}
 
-Même processus que pour détachement du domaine. En attente des maquettes.
+Comme le message l'indique, [supprimez votre association Git](#deleteGitAssociation) dans un premier temps avant de modifier votre domaine.
 
-#### Le domaine est le seul du répertoire
+### Détacher un domaine
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md]:
```suggestion
### Détacher un nom de domaine
```

File: pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md [hunk 14]
```diff
@@ -108,157 +111,148 @@ Avant de valider le formulaire d'association de Git, assurez-vous que :
...
 
-Même processus que pour détachement du domaine. En attente des maquettes.
+Dans votre espace client OVHcloud, accédez à l'onglet `Web Cloud`{.action}, puis cliquez sur `Multisite`{.action}. Dans le tableau qui s'affiche, identifiez la ligne correspondant au domaine que vous souhaitez détacher de votre hébergement web OVHcloud. Cliquez sur `...`{.action} puis sur `Détacher le domaine`{.action}. Deux scénarios sont possibles :
 
-### Détacher un domaine de Git
+#### Le domaine n'est pas le seul attaché au même répertoire
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md]:
```suggestion
#### Le nom de domaine n'est pas le seul attaché au même répertoire
```

File: pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md [hunk 15]
```diff
@@ -108,157 +111,148 @@ Avant de valider le formulaire d'association de Git, assurez-vous que :
...
 
-Dans votre espace client OVHcloud, accédez à l'onglet `Web Cloud`{.action}, puis cliquez sur `Multisite`{.action}. Dans le tableau qui s'affiche, identifiez la ligne correspondant au domaine que vous souhaitez détacher de Git. Cliquez sur `...`{.action} puis sur `Détacher le domaine`{.action}. Deux scénarios sont possibles :
+La fenêtre suivante s'affiche.
 
-- Si votre domaine est le seul du répertoire associé à Git, alors le détachement entrainera la suppression de l'association Git pour le domaine et le répertoire.
-- Si d'autres domaines sont rattachés au même répertoire, alors le détachement entrainera la suppression de l'association Git pour ce domaine.
+![Multisite](/pages/assets/screens/control_panel/product-selection/web-cloud/web-hosting/multisite/detach-domain-popup.png){.thumbnail}
 
-#### Le domaine n'est pas le seul du répertoire
+Cliquez sur `Confirmer`{.action} pour validez le détachement de votre domaine.
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md]:
```suggestion
Cliquez sur `Confirmer`{.action} pour validez le détachement de votre nom de domaine.
```

File: pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md [hunk 16]
```diff
@@ -108,157 +111,148 @@ Avant de valider le formulaire d'association de Git, assurez-vous que :
...
 
-Attente des maquettes
+#### Le domaine est le seul attaché au répertoire
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md]:
```suggestion
#### Le nom de domaine est le seul attaché au répertoire
```

File: pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md [hunk 17]
```diff
@@ -108,157 +111,148 @@ Avant de valider le formulaire d'association de Git, assurez-vous que :
...
 
-#### Le domaine est le seul du répertoire
+La fenêtre suivante s'affiche.
 
---Screenshot placeholder--
+![Multisite](/pages/assets/screens/control_panel/product-selection/web-cloud/web-hosting/multisite/detach-alone-domain.png){.thumbnail}
 
-Cochez la case `J'ai compris, je souhaite détacher mon nom de domaine`{.action} puis appuyez sur `Confirmer`{.action}.
+Comme le message l'indique, [supprimez votre association Git](#deleteGitAssociation) dans un premier temps avant de détacher votre domaine.
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md]:
```suggestion
Comme le message l'indique, [supprimez votre association Git](#deleteGitAssociation) dans un premier temps avant de détacher votre nom de domaine.
```

File: pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md [hunk 18]
```diff
@@ -108,157 +111,148 @@ Avant de valider le formulaire d'association de Git, assurez-vous que :
...
 
 ### Configurer Git
 
 Dans votre espace client OVHcloud, accédez à l'onglet `Web Cloud`{.action}, puis cliquez sur `Multisite`{.action}. Dans le tableau qui s'affiche, identifiez la ligne correspondant au répertoire que vous voulez configurer avec Git. Cliquez sur `...`{.action} puis sur `Configurer Git`{.action}.
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md]:
```suggestion
Dans votre espace client OVHcloud, accédez à l'onglet `Web Cloud`{.action}, puis cliquez sur `Multisite`{.action}. Dans le tableau qui s'affiche, identifiez la ligne correspondant au répertoire que vous voulez configurer avec Git. Cliquez sur le bouton `...`{.action} puis sur `Configurer Git`{.action}.
```

File: pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md [hunk 19]
```diff
@@ -108,157 +111,148 @@ Avant de valider le formulaire d'association de Git, assurez-vous que :
...
 
 ### Informations du dernier déploiement
 
-Après avoir déployé votre site web sur Git, vous pouvez consulter les informations sur le déploiement, comme les erreurs, les tests ou toute information utile. Dans votre espace client OVHcloud, accédez à l'onglet `Web Cloud`{.action}, puis cliquez sur `Multisite`{.action}. Dans le tableau qui s'affiche, identifiez la ligne correspondant au domaine dont vous souhaitez consulter les logs du dernier déploiement. A droite de la ligne, cliquez sur `...`{.action} puis sur `informations du dernier déploiement`{.action}.
+Après avoir déployé votre dépôt Github sur votre hébergement web, vous pouvez consulter les informations sur le dernier déploiement, comme les erreurs, les tests ou toute information utile. Dans votre espace client OVHcloud, accédez à l'onglet `Web Cloud`{.action}, puis cliquez sur `Multisite`{.action}. Dans le tableau qui s'affiche, identifiez la ligne correspondant au domaine dont vous souhaitez consulter les logs du dernier déploiement. À droite de la ligne, cliquez sur `...`{.action} puis sur `informations du dernier déploiement`{.action}.
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md]:
```suggestion
Après avoir déployé votre dépôt Github sur votre hébergement web, vous pouvez consulter les informations sur le dernier déploiement, comme les erreurs, les tests ou toute information utile. Dans votre espace client OVHcloud, accédez à l'onglet `Web Cloud`{.action}, puis cliquez sur `Multisite`{.action}. Dans le tableau qui s'affiche, identifiez la ligne correspondant au domaine dont vous souhaitez consulter les logs du dernier déploiement. À droite de la ligne, cliquez sur le bouton `...`{.action} puis sur `informations du dernier déploiement`{.action}.
```

File: pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md [hunk 20]
```diff
@@ -108,157 +111,148 @@ Avant de valider le formulaire d'association de Git, assurez-vous que :
...
 
---Screenshot placeholder--
+![Multisite](/pages/assets/screens/control_panel/product-selection/web-cloud/web-hosting/multisite/informations-last-git-deployment-button.png){.thumbnail}
 
 Retrouvez sur cet écran toutes les informations relatives au dernier déploiement.
 
-### Supprimer l'association de Git
+### Supprimer l'association de Git <a name="deleteGitAssociation"></a>
+
+Dans votre espace client OVHcloud, accédez à l'onglet `Web Cloud`{.action}, puis cliquez sur `Multisite`{.action}. Dans le tableau qui s'affiche, identifiez la ligne correspondant au répertoire dont vous souhaitez supprimer l'association avec Git. Cliquez sur `...`{.action} puis sur `Supprimer Git`{.action}.
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md]:
```suggestion
Dans votre espace client OVHcloud, accédez à l'onglet `Web Cloud`{.action}, puis cliquez sur `Multisite`{.action}. Dans le tableau qui s'affiche, identifiez la ligne correspondant au répertoire dont vous souhaitez supprimer l'association avec Git. Cliquez sur le bouton `...`{.action} puis sur `Supprimer Git`{.action}.
```

File: pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md [hunk 21]
```diff
@@ -108,157 +111,148 @@ Avant de valider le formulaire d'association de Git, assurez-vous que :
...
 
-Dans votre espace client OVHcloud, accédez à l'onglet `Web Cloud`{.action}, puis cliquez sur `Multisite`{.action}. Dans le tableau qui s'affiche, identifiez la ligne correspondant au répertoire dont vous souhaitez supprimer l'association avec Git. Cliquez sur `...`{.action} puis sur `Supprimer Git`{.action}. La fenêtre suivante s'affiche.
+La fenêtre suivante s'affiche.
 
---Screenshot placeholder--
+![Multisite](/pages/assets/screens/control_panel/product-selection/web-cloud/web-hosting/multisite/delete-git-association-popup.png){.thumbnail}
 
-Le message liste tous les domaines liés au répertoire, vous indiquant que l'association de Git sera supprimée pour tous ces domaines. Cochez la case `Souhaitez-vous vider le contenu du répertoire <votre_repertoire>`{.action} si vous voulez également supprimer le contenu (dossiers et fichiers) du répertoire.
+Le message vous informe que la suppression s'appliquera sur l'ensemble des domaines attachés à votre répertoire. Cochez la case `Souhaitez-vous vider le contenu du répertoire <votre_repertoire>`{.action} si vous voulez également supprimer le contenu (dossiers et fichiers) du répertoire.
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md]:
```suggestion
Le message vous informe que la suppression s'appliquera sur l'ensemble des noms de domaine attachés à votre répertoire. Cochez la case `Souhaitez-vous vider le contenu du répertoire <votre_repertoire>`{.action} si vous voulez également supprimer le contenu (dossiers et fichiers) du répertoire.
```

File: pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md [hunk 22]
```diff
@@ -268,16 +262,16 @@ Cliquez sur `Confirmer`{.action} pour valider la suppression de l'association Gi
 
 > [!primary]
 >
-> Si vous êtes déjà dans le formulaire d'association de Git, copiez l'URL de webhook et passez à l'étape [Configurer le webhook](#configurer-le-webhook)
+> Si vous êtes déjà dans le formulaire d'association de Git, copiez l'URL du webhook et passez à l'étape «[Configurer le webhook](#configureWebhook)»
 >
 
 Dans votre espace client OVHcloud, accédez à l'onglet `Web Cloud`{.action}, puis cliquez sur `Multisite`{.action}. Dans le tableau qui s'affiche, identifiez la ligne correspondant au répertoire sur lequel vous voulez configurer un webhook. Cliquez sur `...`{.action} puis sur `Configurer Git`{.action}.
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md]:
```suggestion
Dans votre espace client OVHcloud, accédez à l'onglet `Web Cloud`{.action}, puis cliquez sur `Multisite`{.action}. Dans le tableau qui s'affiche, identifiez la ligne correspondant au répertoire sur lequel vous voulez configurer un webhook. Cliquez sur le bouton `...`{.action} puis sur `Configurer Git`{.action}.
```

File: pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md [hunk 23]
```diff
@@ -99,7 +102,7 @@ Retournez sur le formulaire d'association de Git de votre espace client OVHcloud
...
 
-En bas du formulaire d'association de Git, une section `Configuration du déploiement automatique`{.action} s'affiche, accompagnée de l'URL de webhook. Configurer un webhook permet à votre dépôt GitHub de notifier automatiquement votre hébergement OVHcloud des événements qui se produisent sur le dépôt Github (nouveau déploiement, changement dans le code, etc.). Cette fonctionnalité est particulièrement utile si vous travaillez en groupe sur le même projet. Pour en savoir plus, découvrez comment [configurer un webhook sur GitHub](#lienverswebhooksection).
+En bas du formulaire d'association de Git, une section `Configuration du déploiement automatique`{.action} s'affiche, accompagnée de l'URL de webhook. Configurer un webhook permet à votre dépôt GitHub de notifier automatiquement votre hébergement OVHcloud des événements qui se produisent sur le dépôt Github (nouveau déploiement, changement dans le code, etc.). Cette fonctionnalité est particulièrement utile si vous travaillez en groupe sur le même projet et que vous souhaitez restez à jour de toutes les modifications du dépôt Github. Pour en savoir plus, découvrez comment [configurer un webhook sur GitHub](#configureWebhook).
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md]:
```suggestion
En bas du formulaire d'association de Git, une section `Configuration du déploiement automatique`{.action} s'affiche, accompagnée de l'URL de webhook. Configurer un webhook permet à votre dépôt GitHub de notifier automatiquement votre hébergement web OVHcloud des événements qui se produisent sur le dépôt Github (nouveau déploiement, changement dans le code, etc.). Cette fonctionnalité est particulièrement utile si vous travaillez en groupe sur le même projet et que vous souhaitez restez à jour de toutes les modifications du dépôt Github. Pour en savoir plus, découvrez comment [configurer un webhook sur GitHub](#configureWebhook).
```

File: pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md [hunk 24]
```diff
@@ -108,157 +111,148 @@ Avant de valider le formulaire d'association de Git, assurez-vous que :
...
+Dans le tableau de l'onglet `Multisite`{.action}, identifiez les lignes correspondant au répertoire que vous voulez associer à Git. Dans la colonne `Git`, si `Erreur` apparaît, cela signifie qu'au moins une des erreurs suivantes est survenue :
 
-Une fois l'erreur identifiée, associez Git à nouveau. Recommencez l'opération en cliquant sur `...`{.action} de la ligne correspondante, puis sur `Associer Git`{.action}.
+- La clé SSH n'a pas été enregistrée dans votre compte Github.
+- Le répertoire d'installation n'est pas vide.
+- L'adresse du dépôt Github n'existe pas ou est erronée.
+- La branche du dépôt Github n'existe pas ou son nom est erroné.
 
-##### Erreurs liées à la clé SSH
+Pour connaître la cause exacte de l'erreur, consultez les informations du dernier déploiement. Dans le tableau, identifiez la ligne correspondant au domaine dont vous souhaitez consulter les logs du dernier déploiement. À droite de la ligne, cliquez sur `...`{.action} puis sur `Informations du dernier déploiement`{.action}.
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/git_integration_webhosting/guide.fr-fr.md]:
```suggestion
Pour connaître la cause exacte de l'erreur, consultez les informations du dernier déploiement. Dans le tableau, identifiez la ligne correspondant au nom de domaine dont vous souhaitez consulter les logs du dernier déploiement. À droite de la ligne, cliquez sur le bouton `...`{.action} puis sur `Informations du dernier déploiement`{.action}.
```

**Review by Kazarus4 — APPROVED** [review]

**Review by OVH-Team-Guides — APPROVED** [review]

**Comment by OVH-Team-Guides** [comment]
Thank you for your contribution @fabienOVH

**Pull request merged**